doc/content/integrations/webhooks/webhook-templates/adding_webhook_templates.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Adding Webhook Templates"
+description: ""
+weight: 4
+---
+
+{{% tts %}} uses webhook templates from the [`lorawan-webhook-templates` Github repository](https://github.com/TheThingsNetwork/lorawan-webhook-templates/).
+
+Once you have created a new webhook template with a proper [format]({{< ref "/integrations/webhooks/webhook-templates/format.md" >}}), you can easily build it locally by following next steps:
+
+1. Clone the [`lorawan-webhook-templates` Github repository](https://github.com/TheThingsNetwork/lorawan-webhook-templates/) to a local folder.
+
+2. Store your webhook template in previously mentioned folder.
+
+3. Update your {{% tts %}} configuration by adding following lines:
+
+```yaml
+as:
+  webhooks:
+    templates:
+      directory: "path-to-the-folder-containing-your-webhook-template"
+```
+
+At this point, you can test your webhook template by [instantiating]({{< ref "/integrations/webhooks/webhook-templates/instantiation.md" >}}) it.
```

Comment:
Instantiation is done by the Console - I'd recommend adding a picture saying 'you can now see the template in the Console and can test it there'.